cli/src/clap_app.rs

```diff
@@ -67,16 +67,19 @@ pub fn get_clap_app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> A
                     "processed",
                     "confirmed",
                     "finalized",
-                    "recent", // Deprecated as of v1.5.5
-                    "single", // Deprecated as of v1.5.5
+                    "recent",       // Deprecated as of v1.5.5
+                    "single",       // Deprecated as of v1.5.5
                     "singleGossip", // Deprecated as of v1.5.5
-                    "root", // Deprecated as of v1.5.5
-                    "max", // Deprecated as of v1.5.5
+                    "root",         // Deprecated as of v1.5.5
+                    "max",          // Deprecated as of v1.5.5
                 ])
                 .value_name("COMMITMENT_LEVEL")
                 .hide_possible_values(true)
                 .global(true)
-                .help("Return information at the selected commitment level [possible values: processed, confirmed, finalized]"),
+                .help(
+                    "Return information at the selected commitment level [possible values: \
+                     processed, confirmed, finalized]",
```

Comment:
nit: Put this on its' own line ?
```
[possible values: processed, confirmed, finalized]",
```
Was going to ask whey we didn't utilize `.possible_values()` but then I scrolled up in the diff 😅

Reply:
Haha, yeah... 😢